e2etest/zt_resume_test.go

```diff
@@ -125,3 +128,52 @@ func TestResume_PublicSource_ContainerTarget(t *testing.T) {
 		EAccountType.Standard(), EAccountType.Standard(), "",
 	)
 }
+
+// TestResume_ConsistentLogs_PublicSource_BlobTarget validates log file consistency for a job initially initiated as a copy operation and later resumed.
+// During the copy operation, a jobId is created and associated with its log file. If a resume operation follows, the same jobId from the copy is used for the log file, and the resumed logs are appended to it.
+// The test ensures that at most two log files are generated: one for scanning logs and one for normal logs.
+
+func TestResume_ConsistentLogs_PublicSource_BlobTarget(t *testing.T) {
```

Comment:
It would be good to validate the contents of the log. Is it possible for you to programmatically open the log file and do some validation?